.github/workflows/modules-integration-test.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: Module Tests
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x]
+        go-version: [1.16.x]
```

Comment:
See https://github.com/google/ko/pull/405

Reply:
Do you need to steal the change I made to the script's `go get`?